Jenkinsfile

```diff
@@ -87,10 +88,10 @@ elifePipeline {
                     sh "IMAGE_TAG=${commit} NODE_ENV=production NODE_CONFIG_ENV=unit-test docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_app_test app bash -c 'scripts/pipeline-log-filter-test.sh'"
                 }, 'test:browser', commit)
             } finally {
-                archiveArtifacts artifacts: "build/screenshots/**/*,build/logs/**/*", allowEmptyArchive: true
-                sh "aws --endpoint-url='http://localhost:4569' s3 ls"
+                archiveArtifacts artifacts: "build/screenshots/**/*,build/logs/**/*,build/*.zip", allowEmptyArchive: true
                 sh "aws --endpoint-url='http://localhost:4569' s3 ls s3://test --recursive"
                 sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml down -v"
+                sh "sudo rm -rf ./build/*"
```

Comment:
`sudo` necessary due to lack of ownership of the files written by the `app` Docker container

Reply:
(I think)

Reply:
The logs for the app container are empty - I think there is a question of whether we should be running the browser tests with the app in the env `DEBUG=*` still think we should be using `NODE_CONFIG=production` ... What do others (@Cooryd @erezmus @will-byrne ) think about this?